Northern Britain and Picts Overhaul

MaelgwnGwynedd opened the pull request.

TurtleShroom commented on WTWSMS/common/religions/00_religions.txt:
```diff
@@ -367,9 +367,11 @@ christian = {
 		can_call_crusade = no
 		priests_can_inherit = no
-		max_wives = 4
+		max_wives = 1
```
Why did you change the wives here? Why don't Insular Christians have three wives?

MaelgwnGwynedd replied:
They didn't practice polygamy. And it's one wife, not two wives.

loup99 replied:
According to input we received they did practice it.

MaelgwnGwynedd replied:
That's wrong, what input?

loup99 replied:
I don't remember Enlil's answer. Will see if I can find it.

MaelgwnGwynedd replied:
Thanks. I think they may have based it on David Hughes' fictional "Chronicles of Britain".

loup99 replied:
This? https://www.amazon.fr/British-Chronicles-David-Hughes/dp/0788433016

loup99 replied:
I found the comment:

> In early Ireland polygamy was practiced using the justification that it was in the old testament.

MaelgwnGwynedd replied:
There's nothing to substantiate that. Hell, the earliest Irish laws stress that a man has only one wife and that any lovers/mistresses he may have don't enjoy the same rights (such as divorce if he should beat her, be accused of adultery or a homosexual).

loup99 replied:
So you would say neither polygamy nor concubinage?

MaelgwnGwynedd replied:
Also, I find it odd they'd ever use a bible as justification when for several centuries the Irish took pride in being insulars. The "Romanii", a faction of Irish monks in the 800's who advocated following the Roman Catholics a lot more, wanted to put an end to incestuous marriage between cousins, and the Irish told them to, well, fuck off.

MaelgwnGwynedd replied:
There's SOME evidence of the Ancient Gaels (Not at ALL the Ancient Britons) had some sort of Polygamy. But we're talking about the 1000's BC here, when they were still glorified hunter gatherers, and the Irish chieftain may have had a harem. That was long dead when the Romans arrived.

Though it's true the Irish practiced what you may call "pseudo-polygamy". They very often divorced (or killed) their wives for dumb reasons just to get a new one, which you can easily see when some Irish kings had six or seven wives in succession. Think of situations like the later Henry VIII. But even _that_ pseudo-polygamy is later period than the mod.

MaelgwnGwynedd replied:
"So you would say neither polygamy nor concubinage?"

Definitely. The Christian "lovers" system is absolutely perfect.

loup99 replied:
And concubines?

loup99 replied:
OK.

TurtleShroom commented on WTWSMS/common/religions/00_religions.txt:
```diff
@@ -1187,6 +1189,46 @@ christian = {
+		pc_marriage = yes
+		religious_clothing_head = 1
+		religious_clothing_priest = 1
+
```
Why do Belanists commit incest and dress like Insular Christians?

MaelgwnGwynedd replied:
Because they're still Christians. Incest was relatively taboo amongst the insulars but not to people of looser faiths like paganism or heresies (De Excidio by Gildas)

loup99 replied:
That makes sense, but there is a difference from it being possible and actively encouraged.

MaelgwnGwynedd replied:
True, but it seemed backwards not to allow it.

MaelgwnGwynedd replied:
See below.

TurtleShroom commented on WTWSMS/common/religions/00_religions.txt:
```diff
@@ -2516,12 +2558,13 @@ pagan_group = {
 		reformed = celtic_pagan_reformed
 		aggression = 1.5 # AI aggression factor
+		bs_marriage = yes
+		pc_marriage = yes
```
Why are unreformed Celtic Pagans incestous? -and if they are incestors, why aren't the Reformed Celtic Pagans also incestous?

MaelgwnGwynedd replied:
See above. Reformed Celtic Pagans never existed, so I can't answer that.

TurtleShroom replied:
Shouldn't Reformed Celtics have the sake  incest ability as normal Celtics?

loup99 replied:
See above on incest and concubines.

MaelgwnGwynedd replied:
They did practice incest.

Maelgwn Gwynedd married his cousin Gwallwyn (Gildas' De Excidio)
Geraint ap Erbin married his unnamed niece (Harleian Genealogies)
Gworthigern (Vortigern) married his daughter
Vortipor married his sister and had incestuous relations with his daughter (Gildas' De Excidio)

I don't think it's not too much of a stretch to allow pc and bs incest for everyone.

MaelgwnGwynedd replied:
Oh, Maelgwn also then went on to marry his nephew's wife. While not blood incest, this was *probably* (and this IS conjectural) political incest, given the Celts' views on the family as a single unit, his niece had become "part of the family" despite not being part of it by blood.
If you see what I mean?

MaelgwnGwynedd replied:
This whole attitude may have been the faults of the Romans, who had very relaxed views on incest (Caligula married his sister and declared her his Goddess sister-wife). It was not actually a crime, but was often used in a slanderous manner.

I can confirm neither of the four men above were slandered for their choice of partner (quasi-nuclear relatives) but only by their other actions. Gildas casually mentions Vortipor's incest then says he's evil since he usurped the throne.

MaelgwnGwynedd replied:
If you believe this is still insufficient grounds to add it, though, could we maybe compromise in some way? Since there is clearly historical evidence of nuclear incest which was quasi-accepted.

loup99 replied:
It being allowed makes sense, although I'm worried that the AI would start prioritising it.

MaelgwnGwynedd replied:
Somewhat how like Xwedodah-style faiths prioritise sister-wives for rulers? That could be a problem.

loup99 replied:
Yes. But I'm not sure if it is another weight that makes that important or if it tied to enabling it.

TurtleShroom replied:
IIRC, they only priorotize it if it is a Xaowada union or it is the most prestigious choice.

loup99 replied:
If it does that it is inappropriate. From what I understand these marriages happened but were not a priority or something which was desired.

MaelgwnGwynedd replied:
Yes. Can we override the priority?

MaelgwnGwynedd replied:
I thought only the divine_bloodline = yes functioned like that.

loup99 replied:
The wiki seems to agree with you: http://www.ckiiwiki.com/Religion_modding

MaelgwnGwynedd commented:
-They didn't practice polygamy. And it's one wife, not two wives.
-Because they're still Christians. Incest was relatively taboo amongst the insulars but not to people of looser faiths like paganism or heresies (De Excidio by Gildas)
-See above. Reformed Celtic Pagans never existed, so I can't answer that.

MaelgwnGwynedd commented:
Those aren't even on this pull request, though.

TurtleShroom commented:
"Max wives" covers ADDITIONAL wives. If you want to keep them from polygamy, you have to remove that line entirely. Right now, the code creates two wives.
 
Why did Insular Christians have polygamy in this Mod? What about concubinage?

MaelgwnGwynedd commented:
I don't anticipate EVERY change. If you want them to have incest, add it.

I don't know why Insular Christians have polygamy in the Mod, all I know is it's wrong. Why do African Warlords have Denmark or why is Fib called "Taezuli"?

MaelgwnGwynedd commented:
Also, they aren't strange additions.

loup99 commented on WTWSMS/common/religions/00_religions.txt:
```diff
@@ -367,9 +367,11 @@ christian = {
 		
 		can_retire_to_monastery = yes
+		bs_marriage = yes
+		pc_marriage = yes
```
Did they practice incest?

MaelgwnGwynedd replied:
See my other comments.

loup99 replied:
What other comment? I posted below on incest.

MaelgwnGwynedd commented:
What input? It's wrong.

MaelgwnGwynedd commented:
What? Just read what I posted.

On 22 Dec 2016 13:33, "Loup" <notifications@github.com> wrote:

> *@loup99* commented on this pull request.
> ------------------------------
>
> In WTWSMS/common/religions/00_religions.txt
> <https://github.com/loup99/WtWSMS/pull/191>:
>
> >  		
>  		can_retire_to_monastery = yes
> +		bs_marriage = yes
> +		pc_marriage = yes
>
> What other comment? I posted below on incest.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/loup99/WtWSMS/pull/191>, or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AUTykWYLrFfeot17f2XZnclhedWFM-6Oks5rKm4FgaJpZM4LJ9Hm>
> .
>

loup99 commented:
Never mind.

MaelgwnGwynedd commented:
Right

MaelgwnGwynedd commented:
Adding some new commits.
-Danish and Dano-Jutish legacies
-Corrected Saxony, Jutland and Angeln
-Renamed them accordingly
-Added the Danish kings who took over Angeln and Jutland
-Added the Anglian (Slevsig and Fyn) and Jutish (Jylland) Kings

MaelgwnGwynedd commented:
How long until you're done reviewing?

loup99 commented:
Depends. The map overhaul and melting-pots are higher priorities for me.

Do you know if any DLC was used for the new portraits?

MaelgwnGwynedd commented:
Celtic DLC, of course.

On 30 Dec 2016 12:37, "Loup" <notifications@github.com> wrote:

> Depends. The map overhaul and melting-pots are higher priorities for me.
>
> Do you know if any DLC was used for the new portraits?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/loup99/WtWSMS/pull/191#issuecomment-269761990>, or mute
> the thread
> <https://github.com/notifications/unsubscribe-auth/AUTykf9BDSJh2EEc8q_rfrd76NupgOU5ks5rNOzcgaJpZM4LJ9Hm>
> .
>

MaelgwnGwynedd commented:
What do you mean the melting-pots?

On 30 Dec 2016 12:37, "Loup" <notifications@github.com> wrote:

> Depends. The map overhaul and melting-pots are higher priorities for me.
>
> Do you know if any DLC was used for the new portraits?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/loup99/WtWSMS/pull/191#issuecomment-269761990>, or mute
> the thread
> <https://github.com/notifications/unsubscribe-auth/AUTykf9BDSJh2EEc8q_rfrd76NupgOU5ks5rNOzcgaJpZM4LJ9Hm>
> .
>

loup99 commented:
OK, then that means I will have to do some modifications to what you currently have. As the files are now, we can't distribute them. 

The melting-pots as in this: https://en.wikipedia.org/wiki/Melting_pot

MaelgwnGwynedd commented:
Why? It can only be accessed by people who have the DLC.

On 30 Dec 2016 17:16, "Loup" <notifications@github.com> wrote:

> OK, then that means I will have to do some modifications to what you
> currently have. As the files are now, we can't distribute them.
>
> The melting-pots as in this: https://en.wikipedia.org/wiki/Melting_pot
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/loup99/WtWSMS/pull/191#issuecomment-269789754>, or mute
> the thread
> <https://github.com/notifications/unsubscribe-auth/AUTykb2l0yIsN52SZ7RivWu14q_YLwLuks5rNS5HgaJpZM4LJ9Hm>
> .
>

loup99 commented:
See the sub-mods

MaelgwnGwynedd commented:
Ok, is the rest of the branch fine?

On 30 Dec 2016 17:57, "Loup" <notifications@github.com> wrote:

> See the sub-mods
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/loup99/WtWSMS/pull/191#issuecomment-269794621>, or mute
> the thread
> <https://github.com/notifications/unsubscribe-auth/AUTykSj77FhcjQmMDtYMHdtLMO9lAbgAks5rNTf-gaJpZM4LJ9Hm>
> .
>

loup99 commented:
That is what I'm going to look at before merging.

MaelgwnGwynedd commented:
But if I keep committing more and more, this will never end.

loup99 commented:
Well, the map overhaul isn't getting done for a long while. But the British Isles will be our step once we are done with Egypt and Hispania. So we will determine exactly how we integrate it depending on the scale of the changes. But don't worry, nothing will get lost, and everything which isn't integrated will be discussed thoroughly beforehand.

MaelgwnGwynedd commented:
Good.

On 31 Dec 2016 08:25, "Loup" <notifications@github.com> wrote:

> Well, the map overhaul isn't getting done for a long while. But the
> British Isles will be our step once we are done with Egypt and Hispania. So
> we will determine exactly how we integrate it depending on the scale of the
> changes. But don't worry, nothing will get lost, and everything which isn't
> integrated will be discussed thoroughly beforehand.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/loup99/WtWSMS/pull/191#issuecomment-269853802>, or mute
> the thread
> <https://github.com/notifications/unsubscribe-auth/AUTykUC_KgnYIxdNzLcrIGk66ZzaNcDvks5rNgNrgaJpZM4LJ9Hm>
> .
>

MaelgwnGwynedd commented:
New additions.

loup99 commented:
The warrior government has already been discussed, so I refer to that for my thoughts.

MaelgwnGwynedd commented:
Ok

On 11 Jan 2017 06:56, "Loup" <notifications@github.com> wrote:

> The warrior government has already been discussed, so I refer to that for
> my thoughts.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/loup99/WtWSMS/pull/191#issuecomment-271785052>, or mute
> the thread
> <https://github.com/notifications/unsubscribe-auth/AUTykbP88LryMxL_hL0tGJ93WQlVaX4hks5rRG7xgaJpZM4LJ9Hm>
> .
>

MaelgwnGwynedd commented:
Don't push this. You're free to overhaul it yourself if you're not going to understand this push is exactly that.

loup99 commented:
If you prefer this to not be included into the main mod at all rather than after the map overhaul, I respect that.

loup99 commented:
Also, it is "yourselves", as in all developers and all of those submitting feedback. Thought it was important to mention as it is an interactive project.

MaelgwnGwynedd commented:
No, no it isn't interactive.

On 12 Jan 2017 17:21, "Loup" <notifications@github.com> wrote:

> Also, it is "yourselves", as in all developers and all of those submitting
> feedback. Thought it was important to mention as it is an interactive
> project.
>
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly, view it on GitHub
> <https://github.com/loup99/WtWSMS/pull/191#issuecomment-272208390>, or mute
> the thread
> <https://github.com/notifications/unsubscribe-auth/AUTykQBIt7aw0g23TtfMg0cshewYNIreks5rRlMjgaJpZM4LJ9Hm>
> .
>

MaelgwnGwynedd commented:
There's no need for a British map overhaul, the borders are *not wrong*.

On 12 Jan 2017 17:18, "Loup" <notifications@github.com> wrote:

> If you prefer this to not be included into the main mod at all rather than
> after the map overhaul, I respect that.
>
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly, view it on GitHub
> <https://github.com/loup99/WtWSMS/pull/191#issuecomment-272207396>, or mute
> the thread
> <https://github.com/notifications/unsubscribe-auth/AUTykU9C31uEYulDFUfVJ4AjsruJ_JVqks5rRlJdgaJpZM4LJ9Hm>
> .
>

loup99 commented:
The borders in the release version are wrong. On this former pull-request, you worked on improving that setup, but you didn't add any province. Our aim is to add provinces and look over the history. Once that is done I had planned to merge the two overhauls.

MaelgwnGwynedd commented:
They are NOT wrong in my overhaul.

On 12 Jan 2017 19:36, "Loup" <notifications@github.com> wrote:

> The borders in the release version are wrong. On this former pull-request,
> you worked on improving that setup, but you didn't add any province. Our
> aim is to add provinces and look over the history. Once that is done I had
> planned to merge the two overhauls.
>
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly, view it on GitHub
> <https://github.com/loup99/WtWSMS/pull/191#issuecomment-272244168>, or mute
> the thread
> <https://github.com/notifications/unsubscribe-auth/AUTykTxllCNd0e7XQrTvQnfgvwqkdEnaks5rRnKRgaJpZM4LJ9Hm>
> .
>

MaelgwnGwynedd commented:
Use this if you want, but I don't want to be a part of this. Sorry.

On 12 Jan 2017 19:38, "Samuel Rowe" <samdrowe@gmail.com> wrote:

> They are NOT wrong in my overhaul.
>
> On 12 Jan 2017 19:36, "Loup" <notifications@github.com> wrote:
>
>> The borders in the release version are wrong. On this former
>> pull-request, you worked on improving that setup, but you didn't add any
>> province. Our aim is to add provinces and look over the history. Once that
>> is done I had planned to merge the two overhauls.
>>
>> —
>> You are receiving this because you modified the open/close state.
>> Reply to this email directly, view it on GitHub
>> <https://github.com/loup99/WtWSMS/pull/191#issuecomment-272244168>, or mute
>> the thread
>> <https://github.com/notifications/unsubscribe-auth/AUTykTxllCNd0e7XQrTvQnfgvwqkdEnaks5rRnKRgaJpZM4LJ9Hm>
>> .
>>
>

MaelgwnGwynedd commented:
Bring overruled, saying my work is basically worthless, refusing to
implement it now or then and not allowing me to communicate with anyone or
the mod, is not on.

On 12 Jan 2017 19:41, wrote:

> Use this if you want, but I don't want to be a part of this. Sorry.
>
> On 12 Jan 2017 19:38, "Samuel Rowe" <samdrowe@gmail.com> wrote:
>
>> They are NOT wrong in my overhaul.
>>
>> On 12 Jan 2017 19:36, "Loup" <notifications@github.com> wrote:
>>
>>> The borders in the release version are wrong. On this former
>>> pull-request, you worked on improving that setup, but you didn't add any
>>> province. Our aim is to add provinces and look over the history. Once that
>>> is done I had planned to merge the two overhauls.
>>>
>>> —
>>> You are receiving this because you modified the open/close state.
>>> Reply to this email directly, view it on GitHub
>>> <https://github.com/loup99/WtWSMS/pull/191#issuecomment-272244168>, or mute
>>> the thread
>>> <https://github.com/notifications/unsubscribe-auth/AUTykTxllCNd0e7XQrTvQnfgvwqkdEnaks5rRnKRgaJpZM4LJ9Hm>
>>> .
>>>
>>

loup99 commented:
Maelgwyn, unfortunately I can't be everywhere all the time. Modding is only a hobby, and if no one can review the content we can't integrate it. I don't know where you read that your work was worthless or that I refused anything, but that was not my intent in case it was how you felt.

MaelgwnGwynedd commented:
"We won't use it because of the Operation Ptolemy changes we will do, not
your correct and already done work."
"This is a team effort." and then "This is not up for discussion"

You've also refused to discuss important things.

On 12 Jan 2017 19:42, "Loup" <notifications@github.com> wrote:

> Maelgwyn, unfortunately I can't be everywhere all the time. Modding is
> only a hobby, and if no one can review the content we can't integrate it. I
> don't know where you read that your work was worthless or that I refused
> anything, but that was not my intent in case it was how you felt.
>
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly, view it on GitHub
> <https://github.com/loup99/WtWSMS/pull/191#issuecomment-272245714>, or mute
> the thread
> <https://github.com/notifications/unsubscribe-auth/AUTykZv-D8vAY6cD4ro08RSZkHx-oljTks5rRnP9gaJpZM4LJ9Hm>
> .
>

loup99 commented:
Also, who has limited you from communicating? That is not something I'm aware of.

MaelgwnGwynedd commented:
You had me banned from the forum where the mod and Operation Ptolemy are
being worked on. I didn't even know about any map overhaul until now, seven
months in.

On 12 Jan 2017 19:44, "Loup" <notifications@github.com> wrote:

> Also, who has limited you from communicating? That is not something I'm
> aware of.
>
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly, view it on GitHub
> <https://github.com/loup99/WtWSMS/pull/191#issuecomment-272246240>, or mute
> the thread
> <https://github.com/notifications/unsubscribe-auth/AUTykTuSK4j6HLyOhK15RIZq8rRpnY2Dks5rRnR5gaJpZM4LJ9Hm>
> .
>

loup99 commented:
> "We won't use it because of the Operation Ptolemy changes we will do, not
> your correct and already done work."

Operation Ptolemy is, if you missed it, an operation about overhauling the entire province map. Some areas we are less happy with, others more. The British Isles are highly requested, so it is an area we will suggest changes to. I don't see how this conflicts with your work, other than that both will have to be merged.  Nothing is getting thrown into a rubbish bin or just discarded outright, if there is anything we don't keep it will be discussed, and everything we touch in any way will be on commits which can be commented upon. 

What is not up for discussion is to not do Operation Ptolemy, as it has already been started and there is a consensus around it. 

I also have a tendency to avoid repeating things, it becomes tedious for me and I don't have enough time to constantly make sure every single line of code is reviewed. This means that there will be delays.

loup99 commented:
> You had me banned from the forum where the mod and Operation Ptolemy are being worked on.

This has already been addressed and closed.

> I didn't even know about any map overhaul until now, seven months in.

You were initially involved in looking at provinces for the British Isles, before you decided to leave the forum.

TurtleShroom commented:
DUDE! He is NOT forsaking your work! HE IS GOING TO MERGE YOUR CHANGES WITH THE OVERHAUL! Stop acting like he is shutting you down.

TurtleShroom commented:
He WILL include your work AFTER the reconfiguration. He is NOT saying your work is useless. It WILL be in the Mod.

MaelgwnGwynedd commented:
MY MAP IS FUCKING CORRECT.

On 12 Jan 2017 19:49, "Loup" <notifications@github.com> wrote:

> "We won't use it because of the Operation Ptolemy changes we will do, not
> your correct and already done work."
>
> Operation Ptolemy is, if you missed it, an operation about overhauling the
> entire province map. Some areas we are less happy with, others more. The
> British Isles are highly requested, so it is an area we will suggest
> changes to. I don't see how this conflicts with your work, other than that
> both will have to be merged. Nothing is getting thrown into a rubbish bin
> or just discarded outright, if there is anything we don't keep it will be
> discussed, and everything we touch in any way will be on commits which can
> be commented upon.
>
> What is not up for discussion is to not do Operation Ptolemy, as it has
> already been started and there is a consensus around it.
>
> I also have a tendency to avoid repeating things, it becomes tedious for
> me and I don't have enough time to constantly make sure every single line
> of code is reviewed. This means that there will be delays.
>
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly, view it on GitHub
> <https://github.com/loup99/WtWSMS/pull/191#issuecomment-272247769>, or mute
> the thread
> <https://github.com/notifications/unsubscribe-auth/AUTykVPKxIxZYwpkd638Pcdscas-aQm3ks5rRnXXgaJpZM4LJ9Hm>
> .
>

MaelgwnGwynedd commented:
There's no need, since the provinces are correct in my changes. There's no
need to overhaul Britain.

On 12 Jan 2017 21:06, "TURTLESHROOM" <notifications@github.com> wrote:

> He WILL include your work AFTER the reconfiguration. He is NOT saying your
> work is useless. It WILL be in the Mod.
>
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly, view it on GitHub
> <https://github.com/loup99/WtWSMS/pull/191#issuecomment-272268785>, or mute
> the thread
> <https://github.com/notifications/unsubscribe-auth/AUTykX_0faQtt2RnGBFluFBZ-VI7ZtdQks5rRofWgaJpZM4LJ9Hm>
> .
>

loup99 commented:
The discussion ends here. MaelgwnGwynedd prefers to not be a part of this. This isn't a place for heated debate.
